Services/UserService/UserService.cs

```diff
@@ -25,17 +28,21 @@ public UserService(IMapper mapper)
         /// <returns>A list of all users.</returns>
         public async Task<List<UsersDto>> GetAllUsers()
         {
-            return (users.Select(c => _mapper.Map<UsersDto>(c))).ToList();
+            List<Users> user = await _context.Users.ToListAsync();
+
+            return (user.Select(c => _mapper.Map<UsersDto>(c))).ToList();
```

Comment:
need to hide the password in an authentic way not to ignore the password through an auto mapper profile.

Reply:
Done